src/routers/SearchRouter.js

```diff
@@ -0,0 +1,79 @@
+// @flow
```

Comment:
Looks like this file already existed and was just renamed. Neat trick for next time: when you rename a file, use `git mv` that way git will recognize that the file was renamed and just show the changes that were actually made to the file!

Reply:
genius